saves/save_3.json

```diff
@@ -4,10 +4,10 @@
         "player": {
             "name": "Player",
             "stats": {
-                "athletic ability": 53,
-                "charisma": 23,
-                "wisdom": 98,
-                "smartness": 70
+                "athletic ability": 60,
```

Comment:
This is not relevant to the PR, avoid including it in the future

Reply:
ok